test/auth_SUITE.erl

```diff
@@ -72,12 +78,21 @@ mqtt_config(no_ssl_user) ->
 
 init_per_testcase(Testcase, Config) when Testcase == ssl_user_auth_success;
                                          Testcase == ssl_user_auth_failure ->
-    Hostname = re:replace(os:cmd("hostname"), "\\s+", "", [global,{return,list}]),
-    User = "O=client,CN=" ++ Hostname,
-    {ok,_} = rabbit_ct_broker_helpers:rabbitmqctl(Config, 0, ["add_user", User, ""]),
-    {ok, _} = rabbit_ct_broker_helpers:rabbitmqctl(Config, 0, ["set_permissions",  "-p", "/", User, ".*", ".*", ".*"]),
-    Config1 = rabbit_ct_helpers:set_config(Config, [{temp_ssl_user, User}]),
+    Config1 = set_cert_user_on_default_vhost(Config),
     rabbit_ct_helpers:testcase_started(Config1, Testcase);
+init_per_testcase(ssl_user_vhost_success, Config) ->
+    Config1 = set_cert_user_on_default_vhost(Config),
+    User = ?config(temp_ssl_user, Config1),
+    {ok, _} = rabbit_ct_broker_helpers:rabbitmqctl(Config1, 0, ["clear_permissions",  "-p", "/", User]),
```

Comment:
I'd introduce a new helper [like this one](https://github.com/rabbitmq/rabbitmq-ct-helpers/blob/master/src/rabbit_ct_broker_helpers.erl#L1070).